src/code/z_map_exp.c

```diff
@@ -364,8 +364,8 @@ void Minimap_Draw(PlayState* play) {
 
     OPEN_DISPS(play->state.gfxCtx, "../z_map_exp.c", 626);
 
-    if (play->pauseCtx.state < 4) {
-        switch (play->sceneId) {
+    if (play->pauseCtx.state < 3) {
+        switch (play->sceneNum) {
```

Comment:
I think this needs changed here back to `sceneId`, try to merge in the current master branch with your fork.